emoji/emoji_transfer_service.py

```diff
@@ -0,0 +1,20 @@
+from emoji import image_client
+
+def transfer(source_client, destination_client, emoji_name):
+    source_dict = source_client.emoji_dict()
+    destination_dict = destination_client.emoji_dict()
+
+    emoji_path = source_dict.get(emoji_name)
+
+    if emoji_path.startswith("alias:"):
+        """If it's an alias, we want to get the url for the file"""
```

Comment:
don't we really want to upload the original file first (if white-listed/existing on slack) and then make an alias to it?

Reply:
Not really. If our list determines that everyone refers to emoji `foo` as `bar`, we want to upload it as `bar`

Reply:
Regarding the original file, I think we can iterate on this to collect aliases.  e.g. if I have

```py
destionation_dict = {
  'sad_cowboy': 'http://my-url.com',
  'emo_cowboy': 'alias:sad_cowboy'
}
```
we can read this input:

```py
["emo_cowboy", "sad_cowboy"]
```

and eventually call:
```py
image = image_client.get(destination_dict.get('sad_cowboy'))

destination_client.add_emoji('sad_cowboy', image)
destination_client.add_alias('emo_cowboy', 'sad_cowboy')
```

Reply:
If we only specify `'emo_cowboy'` in the input, I don't see the need to add both names.